markdown/events/2022/training-october-2022.md

```diff
@@ -0,0 +1,72 @@
+---
+title: nf-core Training - October 2022
+subtitle: A virtual Nextflow and nf-core training event
+type: workshop
+start_date: '2022-10-03'
+start_time: '03:00 CEST'
+end_date: '2022-10-05'
+end_time: '21:30 CEST'
+location_name: Virtual
+import_typeform: true
+---
+
+# Welcome
+
+Join us October 3-5 for the nf-core training event!
+
+The training is entirely virtual and free and will cover the fundamentals of using Nextflow and nf-core. The training will offer something for all skill levels but will be especially useful for those who are new to Nextflow and the nf-core community, or if you are thinking about joining the [nf-core Hackathon](https://nf-co.re/events/2022/hackathon-october-2022) (October 10-12, 2022) for the first time.
+
+# Schedule
+
+The training will be held across three consecutive days in 2.5-hour sessions (including time for a small break and questions). Because nf-core is a global community we are making the training geographically accessible and will be holding separate sessions for different timezones:
```

Comment:
```suggestion
The training will be held across three consecutive days in 2.5-hour sessions (including time for a small break and questions).

Because nf-core is a global community we are making the training geographically accessible and will be holding separate sessions for different timezones:
```